uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -330,6 +331,23 @@ public void setMqttTimeoutSec(int mqttTimeoutSec) {
      */
     @When("I subscribe {string} to {string} with qos {int}")
     public void subscribe(@NonNull String clientDeviceId, @NonNull String topicFilterString, int qos) {
+        subscribe(clientDeviceId, topicFilterString, qos,
+                Mqtt5RetainHandling.MQTT5_RETAIN_DO_NOT_SEND_AT_SUBSCRIPTION.getNumber());
```

Comment:
Why are you choosing not to receive retained messages? Default here should be to get them.

Reply:
Please use DEFAULT_SUBSCRIBE_RETAIN_HANDLING.
Change value of DEFAULT_SUBSCRIBE_RETAIN_HANDLING.